src/Interpreters/InterpreterCreateQuery.cpp

```diff
@@ -2333,6 +2337,18 @@ void InterpreterCreateQuery::processSQLSecurityOption(ContextPtr context_, ASTSQ
         context_->checkAccess(AccessType::ALLOW_SQL_SECURITY_NONE);
 }
 
+String InterpreterCreateQuery::getRandomSuffix(size_t size)
```

Comment:
Probably we can reuse this function
https://github.com/ClickHouse/ClickHouse/blob/master/src/Common/getRandomASCIIString.cpp#L8